apps/sms/views/conversation/js/conversation.js

```diff
@@ -80,6 +80,11 @@ var ConversationView = {
   showErrorInFailedEvent: '',
   previousSegment: 0,
 
+  /**
+   * Set to true after init is run.
+   */
+  _isReady: false,
```

Comment:
I like that init can be run repeatedly without needing to reset state -- and we'd need to do it if we'd use an early return. This is useful to test more easily.

Here we need to reset _isReady but only for the tests that specific test this state. Otherwise we don't care.

So that's why I did it like this. But anyway I don't really like all the lazy loading part for a long time and for sure I'd be open to do that again later. For example the "Navigation" object could keep the loading status of the panels in the VIEWS object, and as you said call init itself after lazy loading the JS file (or through a wrapper). There are so many things we can do better.